tensorflow_addons/layers/embedding_bag.py

```diff
@@ -49,8 +49,13 @@ def _embedding_bag(
     Returns:
       A `Tensor` of the format specified by `data_format`.
     """
-    if weights is None:
+    if weights is None and combiner == "sum":
         weights = tf.ones_like(indices, dtype=params.dtype)
+    elif weights is None and combiner == "mean":
+        weights = tf.ones_like(indices, dtype=params.dtype) / tf.cast(
+            tf.shape(indices)[1], params.dtype
+        )
+        combiner = "sum"
```

Comment:
Why we have this combiner overriding?

Reply:
It was a quick workaround, but the output and grads are correct, and performance is fine! Since combiner == "mean" does not support weights, we can get the right results by creating a dummy weight array (we do that anyway), then scaling it so that we get an unweighted mean instead of an unweighted sum.

Reply:
Maybe you should call ```tf.reduce_mean``` after custom op instead of integrating with it or combiner overriding.